DATA/production/workflow-multiplicities.sh

```diff
@@ -6,21 +6,21 @@ SOURCE_GUARD_MULTIPLICITIES=1
 # ---------------------------------------------------------------------------------------------------------------------
 # Threads
 
-[[ -z $NITSDECTHREADS ]] && NITSDECTHREADS=2
-[[ -z $NMFTDECTHREADS ]] && NMFTDECTHREADS=2
+if [[ -z $NITSDECTHREADS ]]; then NITSDECTHREADS=2; fi
+if [[ -z $NMFTDECTHREADS ]]; then NMFTDECTHREADS=2; fi
 
-[[ -z $NMFTTHREADS ]] && NMFTTHREADS=2
+if [[ -z $NMFTTHREADS ]]; then NMFTTHREADS=2; fi
 
-[[ -z $SVERTEX_THREADS ]] && SVERTEX_THREADS=$(( $SYNCMODE == 1 ? 1 : 2 ))
+if [[ -z $SVERTEX_THREADS ]]; then SVERTEX_THREADS=$(( SYNCMODE == 1 ? 1 : 2 )); fi
 # FIXME: multithreading in the itsmft reconstruction does not work on macOS.
 if [[ $(uname) == "Darwin" ]]; then
     NITSDECTHREADS=1
     NMFTDECTHREADS=1
 fi
 
-[[ $SYNCMODE == 1 ]] && NTRDTRKTHREADS=1
+if [[ $SYNCMODE == 1 ]]; then NTRDTRKTHREADS=1; fi
 
-[[ -z $NGPURECOTHREADS ]] & NGPURECOTHREADS=-1 # -1 = auto-detect
```

Comment:
The typo here always forced auto-detect I think. Not sure if you have used external `NGPURECOTHREADS` in the past for some checks @davidrohr ?

Reply:
thx for spotting, that was never used indeed!